piker/clearing/_client.py

```diff
@@ -194,21 +196,22 @@ async def open_ems(
     - 'dark_executed', 'broker_executed'
     - 'broker_filled'
 
-    """
+    '''
     # wait for service to connect back to us signalling
     # ready for order commands
     book = get_orders()
 
+    from ..data._source import uncons_fqsn
+    broker, symbol, suffix = uncons_fqsn(fqsn)
+
     async with maybe_open_emsd(broker) as portal:
 
         async with (
-
             # connect to emsd
             portal.open_context(
 
                 _emsd_main,
-                broker=broker,
-                symbol=symbol.key,
+                fqsn=fqsn,
```

Comment:
some docs somewhere to explain what `fqsn` stands for would be welcomed

Reply:
yah no kidding 😂 

the description in this PR would probably me more or less sufficient yah?

Reply:
definitely, but it would be nice to have it in the code somewhere

Reply:
added as doc string on `Symbol.front_fqsn()`.